src/lib.rs

```diff
@@ -611,6 +615,25 @@ pub trait Watcher: Sized {
     /// Returns an error in the case that `path` has not been watched or if removing the watch
     /// fails.
     fn unwatch<P: AsRef<Path>>(&mut self, path: P) -> Result<()>;
+
+    /// Configure notify with Configs.
+    fn configure(&self, option: Config) -> Result<()> {
```

Comment:
Thinking more about this...should I make this a collection of Config rather than just a Config?